client/lib/application/src/application-lifecycle/components/Charts.jsx

```diff
@@ -4,19 +4,40 @@ import { Link } from 'react-router';
 import * as Routes from 'application/src/routes';
 import Chart from 'common/src/components/pure/Chart.jsx';
 import Icon from 'react-fa';
-import TitleWithButton from 'common/src/components/pure/TitleWithButton.jsx';
 import ThreeColumns from 'common/src/components/pure/ThreeColumns.jsx';
+import * as utils from './charts_utils.jsx';
+
 
 const CHART_HEIGHT = 200;
 
 const Charts = (props) => {
     const arrayOfChartComponets = props.versions.map((version, index) => {
         const versionDataSet = props.versionDataSets.find(e => e.version_id == version.id);
 
-        const titleWithButton = <TitleWithButton
-            title   = {version.id}
-            onClick = {() => props.onDeselect(version.id)}
-        />;
+        const leftElements =
+            <div style={{display: 'flex', flexDirection: 'column', justifyContent: 'flex-end'}}>
+                <div>
+                    <Link
+                        to={Routes.verApproval({applicationId: props.applicationId, versionId: version.id})}
+                        className='btn btn-default btn-small'>
+                        <Icon name='check' />
+                    </Link>
+                    Approvals
```

Comment:
Please make sure the buttons and text align properly (using fixed width icons etc.)